CODEOWNERS

```diff
@@ -6,4 +6,13 @@
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
 
-* @fmvilas @derberg @dalelane @smoya @char0n @asyncapi-bot-eve
\ No newline at end of file
+* @fmvilas @derberg @dalelane @smoya @char0n @asyncapi-bot-eve
+
+
+/bindings/anypointmq/       @GeraldLoeffler
+/bindings/ibmmq/            @rcoppen
+/bindings/kafka/            @lbroudoux @dalelane
+/bindings/googlepubsub/     @whitlockjc
+/bindings/solace/           @damaru-inc @CameronRushton
+/bindings/*.json            @KhudaDad414
+/bindings/pulsar/           @VisualBean
```

Comment:
Just a friendly reminder, you will need to invite all as contributors once it's fully merged 🙂

Reply:
can you also ping these maintainers in https://github.com/asyncapi/bindings/issues/113 so they are aware 🙏🏼

Reply:
Will do once merged 👍